src/diagnostics/land_compute_methods.jl

```diff
@@ -0,0 +1,146 @@
+@land_compute "snow_water_equivalent" BucketModel Y.bucket.σS
+@land_compute "soil_temperature" BucketModel Y.bucket.T
+@land_compute "subsurface_water_storage" BucketModel Y.bucket.W
+@land_compute "surface_water_content" BucketModel Y.bucket.Ws
+
+### SoilCanopyModel ###
+
+# variables stored in p (diagnostics variables stored in the cache)
+
+## Canopy Module ##
+
+# Canopy - Solar Induced Fluorescence
+@land_compute "solar_induced_fluorescence" SoilCanopyModel p.canopy.sif.SIF
+
+# Canopy - Autotrophic respiration
+@land_compute "autotrophic_respiration" SoilCanopyModel p.canopy.autotrophic_respiration.Ra
+
+# Canopy - Conductance
+@land_compute "medlyn_term" SoilCanopyModel p.canopy.conductance.medlyn_term
+@land_compute "stomatal_conductance" SoilCanopyModel p.canopy.conductance.gs
+
+# Canopy - Energy
+@land_compute "canopy_aerodynamic_resistance" SoilCanopyModel p.canopy.energy.r_ae
+@land_compute "canopy_latent_heat_flux" SoilCanopyModel p.canopy.energy.lhf
+@land_compute "canopy_sensible_heat_flux" SoilCanopyModel p.canopy.energy.shf
+
+# Canopy - Hydraulics
+@land_compute "area_index" SoilCanopyModel p.canopy.hydraulics.area_index.leaf
+@land_compute "canopy_water_potential" SoilCanopyModel p.canopy.hydraulics.ψ
+@land_compute "cross_section" SoilCanopyModel p.canopy.hydraulics
+@land_compute "cross_section_roots" SoilCanopyModel p.canopy.hydraulics.fa_roots
+@land_compute "moisture_stress_factor" SoilCanopyModel p.canopy.hydraulics.β
+
+# Canopy - Photosynthesis
+@land_compute "photosynthesis_net_canopy" SoilCanopyModel p.canopy.photosynthesis.GPP
+@land_compute "photosynthesis_net_leaf" SoilCanopyModel p.canopy.photosynthesis.An
+@land_compute "respiration_leaf" SoilCanopyModel p.canopy.photosynthesis.Rd
+@land_compute "vcmax25" SoilCanopyModel p.canopy.photosynthesis.vcmax25
+
+# Canopy - Radiative Transfer
+@land_compute "near_infrared_radiation" SoilCanopyModel p.canopy.radiative_transfer.nir
+@land_compute "near_infrared_radiation_absorbed" SoilCanopyModel p.canopy.radiative_transfer.anir
+@land_compute "near_infrared_radiation_reflected" SoilCanopyModel p.canopy.radiative_transfer.rnir
+@land_compute "near_infrared_radiation_transmitted" SoilCanopyModel p.canopy.radiative_transfer.tnir
+@land_compute "photosynthetically_active_radiation" SoilCanopyModel p.canopy.radiative_transfer.par
+@land_compute "photosynthetically_active_radiation_absorbed" SoilCanopyModel p.canopy.radiative_transfer.apar
+@land_compute "photosynthetically_active_radiation_reflected" SoilCanopyModel p.canopy.radiative_transfer.rpar
+@land_compute "photosynthetically_active_radiation_transmitted" SoilCanopyModel p.canopy.radiative_transfer.tpar
+@land_compute "radiation_longwave_net" SoilCanopyModel p.canopy.radiative_transfer.LW_n
+@land_compute "radiation_shortwave_net" SoilCanopyModel p.canopy.radiative_transfer.SW_n
+
+# Canopy - Transpiration
+@land_compute "canopy_transpiration" SoilCanopyModel p.canopy.transpiration
```

Comment:
If this is the only variable not in a canopy namespace, should we move it to e.g., conductance?

Reply:
we can revisit this, but maybe a second PR? or maybe even plant hydraulics

Reply:
it is actually already in conductance. I am not sure if this was a typo or if it has been moved at some point